<launch>
  <node pkg="frontier_explore" type="masked_costmap.py" name="masked_costmap" output="screen">
    <!-- 该范围其实为盒子坐标范围，范围外的代价地图将被过滤掉，可以考虑稍微放大 -->
    <param name="x_min" value="2" />
    <param name="x_max" value="22" />
    <param name="y_min" value="11" />
    <param name="y_max" value="19" />
  </node>
</launch>